### DISCUSSION TITLE
Scrolling Detection without Layout Management?

### DISCUSSION THREAD

**Discussion by jsprog** [slint-ui/slint#5537] [Q&A]
**What inspired me?**
- The maps example really inspired me to build a better alternative as a custom component(s)

**What I'm trying to achieve?**
- A virtual grid for tiles with infinite scrolling in all directions.
- Tiles are placed automatically to fill the entire window area.
- When scrolling, more tiles may get added to cover any empty space, and other invisible tiles may get removed (could be done without rust)
- Newly added tiles need to request images to be served later by rust code (`request_tile_image`)

**What I really need?**
- I need scrolling detection (`delta_x` and `delta_y`) without layout management to later use the received deltas to add/remove tiles dynamically and request images as needed.
- Of course, scrolling detection should not prevent children events such as gestures, etc...

**Comment by ogoffart** [comment]
You can use a TouchArea on the tile and implement the panning yourself bu handling the pointer event.
